### DISCUSSION TITLE
Post Quantum Crypto in strongSwan 6

### DISCUSSION THREAD

**Discussion by ghost** [strongswan/strongswan#2000] [General]
Please provide a working example of `swantctl.conf` for both `moon` and `carol` with Post Quantum Crypto in strongSwan 6.

The testing procedure documented at https://g800.pages.dev/strongswan6 results in no proposal chosen.

**Comment by tobiasbrunner** [comment]
> The testing procedure documented at https://g800.pages.dev/strongswan6 results in no proposal chosen.

You might want to start by reading the logs on the host that sent that error.
